src/clients/db/Repository.ts

```diff
@@ -57,16 +57,15 @@ export default class Repository<T extends AbstractDBModel> {
       return this.save(entity)
     }
 
-    const { id, rev, ...dataToSave } = entity
-
     try {
-      await this.find(entity.id)
+      const existingEntity = await this.find(entity.id)
+      const { id, rev, ...restOfDoc } = existingEntity
       const entityToUpdate = {
         _id: id,
         _rev: rev,
-        ...dataToSave,
+        ...restOfDoc,
+        ...entity,
```

Comment:
don't remember changing this...

Reply:
yeah accidentally overwrote your fix  for the duplicate patients, should be good now